lib/ronin/recon/values/url.rb

```diff
@@ -19,6 +19,10 @@
 #
 
 require 'ronin/recon/value'
+require 'ronin/recon/values/domain'
+require 'ronin/recon/values/website'
+require 'ronin/recon/values/wildcard'
+require 'ronin/recon/values/host'
```

Comment:
Should probably be using `require_relative` now. The `main` branch has `require_relative`, and I will probably rebase the `0.2.0` branch today.